Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -237,6 +237,15 @@ input[type=submit], input[type=button], input[type=reset], input[type=checkbox],
     cursor: unset;
 }
 
+input[type=submit], input[type=button], input[type=reset] {
+    padding: 4px 8px;
+    background-color: -libweb-palette-button;
+    border: 1px solid -libweb-palette-inactive-window-border1;
+}
+input[type=submit], input[type=button], input[type=reset] {
```

Comment:
These are missing `:hover`

Reply:
Adjusted. 
This actually made me find a bug where it is currently not possible to hover over input elements which have no text value set. This is because they have no content width and the hover is only triggered on hovering over the content, not include the padding. This seems to be unrelated to this as it is also happening for <div>'s, so it should work when that gets fixed.